roles/setup_dbt2/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Gather the pg_cluster_nodes information
+  set_fact:
+    pg_cluster_nodes: "{{ lookup('edb_devops.edb_postgres.pg_sr_cluster_nodes', wantlist=True) }}"
+
+- name: Get the primary private IP address
+  set_fact:
+    primary_private_ip: "{{ node.private_ip }}"
+  loop: "{{ pg_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  when:  node.node_type == 'primary'
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
+- name: Include DBT-2 client systems into pg_hba
+  include_tasks: update_pg_hba.yml
+
+- name: Install packages for DBT-2 (server, yum)
+  yum:
+    name: "{{ dbt2_packages_pg }}"
+  when: os in [ 'CentOS7', 'RHEL7']
+  become: yes
+
+- name: Install packages for DBT-2 (server, dnf)
+  dnf:
+    name: "{{ dbt2_packages_pg }}"
+  when: os in [ 'CentOS8', 'RHEL8']
+  become: yes
+
+- name: Include DBT-2-installation
+  include_tasks: install_dbt2.yml
+
+- name: Install DBT-2 UDF
+  shell:
+    cmd: make USE_PGXS=1 install
+    chdir: /usr/local/src/dbt2/storedproc/pgsql/c
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/usr/pgsql-{{ pg_version }}/bin"
```

Comment:
`USE_PGXS` could be passed as an environment variable, isn't it?

Reply:
Yes, updated.